index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'self' 'unsafe-inline' https://use.typekit.net https://p.typekit.net; script-src 'self' 'unsafe-eval'; connect-src 'self' localhost:* https://allsearch-api.princeton.edu https://allsearch-api-staging.princeton.edu; font-src 'self' https://use.typekit.net; base-uri 'none';img-src 'self';">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'self' 'unsafe-inline' https://use.typekit.net https://p.typekit.net; script-src 'self' 'unsafe-eval'; connect-src 'self' localhost:* https://allsearch-api.princeton.edu https://allsearch-api-staging.princeton.edu https://bibdata.princeton.edu https://bibdata-staging.princeton.edu; font-src 'self' https://use.typekit.net; base-uri 'none';img-src 'self';">
```

Comment:
We need to add bibdata to the content security policy, to let the browser know that it's safe to make these requests.